policy/modules/services/dovecot.te

```diff
@@ -124,6 +124,8 @@ create_files_pattern(dovecot_t, dovecot_var_log_t, dovecot_var_log_t)
 setattr_files_pattern(dovecot_t, dovecot_var_log_t, dovecot_var_log_t)
 logging_log_filetrans(dovecot_t, dovecot_var_log_t, { file dir })
 
+allow dovecot_t dovecot_spool_t:file map;
```

Comment:
This can be combined with the below `manage_files_pattern()` to `mmap_manage_files_pattern()`.